Fix pub/sub stream doc

dzou opened the pull request:
Fix broken snippet in Spring Cloud Stream docs for Pub/Sub.

artemptushkin commented:
@dzou don't you want to update this part as well https://googlecloudplatform.github.io/spring-cloud-gcp/2.0.3/reference/html/index.html#error-handling? This is not up-to-date, not compiled even

dzou commented:
> @dzou don't you want to update this part as well https://googlecloudplatform.github.io/spring-cloud-gcp/2.0.3/reference/html/index.html#error-handling? This is not up-to-date, not compiled even

Yeah, I fixed this in the changes I made to `docs/src/main/asciidoc/spring-integration-pubsub.adoc` in this PR. I changed the parameter to PubSubTemplate instead of PubSubFactory -- is this not correct?

artemptushkin commented:
ah right, sorry missed. That's ok, thanks!
